Update/ura_014.txt

```diff
@@ -4590,26 +4590,26 @@ void main()
 //rそれなのに、…疑心暗鬼に陥るあまりにそばにいる人の気持ちすら疑って、激情に任せて悪しざまに酷いことを言ってしまって…。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "それなのに、…疑心暗鬼に陥るあまりにそばにいる人の気持ちすら疑って、激情に任せて悪しざまに酷いことを言ってしまって…。",
-			NULL, "And yet... even the people closest to her were doubting her, and abandoning her after saying terrible things...", Line_Normal);
+			NULL, "And yet... excess paranoia led her to doubt the feelings of even those closest to her, driving herself into fury and saying horrible things to them...", Line_Normal);
 	ClearMessage();
 
 //r冷静になってからその過ちを悟り、失ったと思ったものの大きさを理解した瞬間、…心をつなぎとめていた支えが、…一気に崩れ落ちてしまったのだ。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "冷静になってからその過ちを悟り、失ったと思ったものの大きさを理解した瞬間、…心をつなぎとめていた支えが、…一気に崩れ落ちてしまったのだ。",
-			NULL, "After he calmed down, he realized his mistake and understood the magnitude of what she'd lost... but the support holding her heart together... fell apart all at once.", Line_Normal);
+			NULL, "So the moment she calmed down, realized her mistake, and understood the magnitude of what she thought she'd lost... the support holding her heart together... fell apart all at once.", Line_Normal);
 	ClearMessage();
 
 //真澄r-vS14/00/VT4Bb_masumi1003.（…もう、謝っても…遅すぎる）
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>真澄</color>", NULL, "<color=#f5e6d3>Masumi</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_masumi1003", 256, TRUE);
 	OutputLine(NULL, "（…もう、謝っても…遅すぎる）",
-			NULL, "(...Even if he apologizes... it's already too late.)", Line_Normal);
+			NULL, "(...Even if I apologize... it's already too late.)", Line_Normal);
 	ClearMessage();
 
 //rだけど、自分の想いだけは信じてほしい。…正気を失ってまともな思考もできなくなった彼女が、最後に切実に願ったその心の現われが…自ら命を絶つことだった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "だけど、自分の想いだけは信じてほしい。…正気を失ってまともな思考もできなくなった彼女が、最後に切実に願ったその心の現われが…自ら命を絶つことだった。",
-			NULL, "But, she only wanted him to believe her thoughts. ...She had lost her senses and couldn't think properly, but the last thing she earnestly wished for... was to end her life.", Line_Normal);
+			NULL, "However, she at least wanted to believe in her feelings. ...The last thing she earnest wished for after losing her sanity and ability to think properly... was to end her life.", Line_Normal);
```

Comment:
Isn't てほしい usually used when someone wants *someone else* to do something?  It might make more sense as some sort of extension of the previous line's thought, would explain the sudden present tense as well

Also earnest**ly**